code/modules/client/preference_setup/general/02_language.dm

```diff
@@ -12,6 +13,10 @@
 	if(islist(pref.alternate_languages))			// Because aparently it may not be?
 		testing("LANGSANI: Loaded from [pref.client]'s character [pref.real_name || "-name not yet loaded-"] savefile: [english_list(pref.alternate_languages || list())]")
 	S["language_prefixes"]	>> pref.language_prefixes
+	//VORE Edit Begin
+	S["species"]			>> pref.species
```

Comment:
Why is species being moved there?

Reply:
Probably a mistaken cherrypick from CHOMP, I can fix that.